src/components/Event/index.js

```diff
@@ -156,8 +156,9 @@ const Event = ({ match, location }) => {
                                         value={tempEvent.topics}
                                         multiline
                                         onChange={handleChange}
-                                    />
+                                        />
                                     <TextField
+                                        style={{ marginBottom: '20px' }}
```

Comment:
nit: `px` is the default unit
```suggestion
                                        style={{ marginBottom: 20 }}
```